### ISSUE TITLE
[Request]: Delete player-spawned NPCs

### ISSUE THREAD

**Issue by snowleopardd** [Yimura/YimMenu#3188]
### Describe the bug

So i was in a lobby and a modder spawned a bunch of planes in god mode and i couldn't get rid of them, not even using the entity remover, is there any way?

### Steps To Reproduce

1. Go to online
2. Make a modder spawn enemy npc's to you

### Expected Behavior

There should be a way to delete them

### Logs

[cout.txt](https://github.com/YimMenu/YimMenu/files/15444321/cout.txt)

### Screenshots / Video

![Screenshot 2024-05-25 140259](https://github.com/YimMenu/YimMenu/assets/88536465/e1ac1da9-ccb6-49df-8267-e421c58c8504)

### Store

Epic Games

### Language

English

### YimMenu Language

English (US)

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### Coload

- [X] I've tested this without co-loading anything.

**Comment by gir489returns** [comment]
This is not a bug.

**Comment by binded-zz** [comment]
i know what your talking about lol 
i have not tried this because its to much bother to even try 
but did you try to use the delete gun idk if it works or not worth a try 
the world delete dont work ;(

**Comment by snowleopardd** [comment]
> i know what your talking about lol i have not tried this because its to much bother to even try but did you try to use the delete gun idk if it works or not worth a try the world delete dont work ;(

Yeah i tried both, using the delete gun it says its not an entity

**Comment by binded-zz** [comment]
ok so this may or may not work did you try sch's lua
theres npc and other controls on that
i find that it even deletes players god mod cars off the map with them  even in it..
theres a english version of it 

https://github.com/sch-lda/SCH-LUA-YIMMENU

https://github.com/Drsexo/English-Sch-lua/
first screen shot shows  the npc and vehicle controls

**Comment by snowleopardd** [comment]
> ok so this may or may not work did you try sch's lua theres npc and other controls on that i find that it even deletes players god mod cars off the map with them even in it.. theres a english version of it
> 
> https://github.com/sch-lda/SCH-LUA-YIMMENU
> 
> https://github.com/Drsexo/English-Sch-lua/ first screen shot shows the npc and vehicle controls

Thank you goat!!, i hope it gets added to yim as native soon

**Comment by binded-zz** [comment]
yeah i love his script wish he had more time to work on it more he has like the most functions that you just dont see in any of the other yim lua scripts 
like the npc, car, controls  the user whitelist and actions list the protections  the debug stuff
but the layout and the fact it takes the whole damn screen with value, boxes that scroll across the whole screen with a value thats not more then 5 chars long but box is 100 wide 

i dont think that they will add all of that stuff to the base it can be pretty toxic if you get around other players setting them all on fire, making them ragdoll, deleting them off the map, and what not
